app/code/Bulbulatory/Recomendations/view/frontend/web/js/view/checkout/summary/customdiscount.js

```diff
@@ -0,0 +1,28 @@
+define(
+    [
+        'jquery',
+        'Magento_Checkout/js/view/summary/abstract-total',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/totals'
+    ],
+    function ($,Component, quote, totals) {
+        "use strict";
+        return Component.extend({
+            defaults: {
+                template: 'Bulbulatory_Recomendations/checkout/summary/customdiscount'
+            },
+            totals: quote.getTotals(),
+
+            isDisplayedCustomdiscount : function(){
+                return true;
```

Comment:
Jeśli zniżki nie ma, to raczej nie chcemy jej pokazywać. Można sprawdzić, czy zniżka jest naliczona (pobrać ją tak, jak w funkcji poniżej i sprawdzić, czy ma jakąś wartość) i na tej podstawie zwrócić true/false.